manageiq-operator/pkg/helpers/miq-components/httpd.go

```diff
@@ -243,8 +243,13 @@ func configureHttpdAuth(spec *miqv1alpha1.ManageIQSpec, podSpec *corev1.PodSpec)
 	}
 }
 
-func httpdImage(namespace, tag string, privileged bool) string {
+func httpdImage(imageref string, namespace string, tag string, privileged bool) string {
```

Comment:
Can this logic be moved into `cr.go`?  Then all callers can use `cr.Spec.HttpdImage`